### PR TITLE
Send analytics for Domains created by a user

### PR DESCRIPTION
**Pull request by biyeun**
@sravfeyn @dannyroberts (cc @TylerSheffels)

### CODE REVIEW

File: corehq/apps/analytics/tasks.py [hunk 1]
```diff
@@ -288,6 +291,10 @@ def track_periodic_data():
                 {
                     'property': 'max_mobile_workers_in_a_domain',
                     'value': max_workers
+                },
+                {
+                    'property': 'project_spaces_created_by_user',
```
Comment (sravfeyn) [inline corehq/apps/analytics/tasks.py]: Is the property `project_spaces_created_by_user ` created in Hubspot [portal](https://github.com/dimagi/commcare-hq/tree/created-domain-analytics/corehq/apps/analytics#addingremoving-hubspot-properties) as well?

File: corehq/apps/analytics/tasks.py [hunk 2]
```diff
@@ -278,6 +279,8 @@ def track_periodic_data():
             if domain in domains_to_mobile_users and domains_to_mobile_users[domain] > max_workers:
                 max_workers = domains_to_mobile_users[domain]
 
+        project_spaces_created = get_domains_created_by_user(email)
+
```
Comment (sravfeyn) [inline corehq/apps/analytics/tasks.py]: Why does this need periodic check, shouldn't it get triggered when a domain gets created?

Comment (biyeun) [inline corehq/apps/analytics/tasks.py]: the spec asked for this to be part of the periodic check

File: corehq/apps/analytics/tasks.py (hunk 1 above)
Comment (biyeun) [inline corehq/apps/analytics/tasks.py]: added, though it looks like it won't accept lists as the value. updating PR

**Comment by biyeun** [comment]
@sravfeyn PR updated

**Pull request merged**